pgcli/pgcompleter.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, smart_completion=True):
 
         self.special_commands = []
         self.databases = []
-        self.dbmetadata = {}
+        self.dbmetadata = {'tables': {}, 'functions': {}}
```

Comment:
Would it make sense to separate this into two different dictionaries?

self.tables_metadata = {}
self.funcs_metadata = {}

I'm not suggesting we should do this, but I'm just thinking out loud. I see a lot of `metadata = self.dbmetadata['tables']` call sprinkled in the code. I wonder if separate vars will help.